discord/__main__.py

```diff
@@ -198,106 +230,147 @@ def newbot(parser, args) -> None:
+            print(f"warning: could not create .gitignore file ({exc})")
+
+    print("successfully made bot at", new_directory)
 
-    print('successfully made bot at', new_directory)
 
 def newcog(parser, args) -> None:
     cog_dir = to_path(parser, args.directory)
     try:
         cog_dir.mkdir(exist_ok=True)
     except OSError as exc:
-        print(f'warning: could not create cogs directory ({exc})')
+        print(f"warning: could not create cogs directory ({exc})")
 
     directory = cog_dir / to_path(parser, args.name)
-    directory = directory.with_suffix('.py')
+    directory = directory.with_suffix(".py")
     try:
-        with open(str(directory), 'w', encoding='utf-8') as fp:
-            attrs = ''
-            extra = _cog_extras if args.full else ''
+        with open(str(directory), "w", encoding="utf-8") as fp:
+            attrs = ""
+            extra = _cog_extras if args.full else ""
             if args.class_name:
                 name = args.class_name
             else:
                 name = str(directory.stem)
-                if '-' in name or '_' in name:
-                    translation = str.maketrans('-_', '  ')
-                    name = name.translate(translation).title().replace(' ', '')
+                if "-" in name or "_" in name:
+                    translation = str.maketrans("-_", "  ")
+                    name = name.translate(translation).title().replace(" ", "")
                 else:
                     name = name.title()
 
             if args.display_name:
                 attrs += f', name="{args.display_name}"'
             if args.hide_commands:
-                attrs += ', command_attrs=dict(hidden=True)'
+                attrs += ", command_attrs=dict(hidden=True)"
             fp.write(_cog_template.format(name=name, extra=extra, attrs=attrs))
     except OSError as exc:
-        parser.error(f'could not create cog file ({exc})')
+        parser.error(f"could not create cog file ({exc})")
     else:
-        print('successfully made cog at', directory)
+        print("successfully made cog at", directory)
+
 
 def add_newbot_args(subparser: argparse._SubParsersAction) -> None:
-    parser = subparser.add_parser('newbot', help='creates a command bot project quickly')
+    parser = subparser.add_parser(
+        "newbot", help="creates a command bot project quickly"
+    )
     parser.set_defaults(func=newbot)
 
-    parser.add_argument('name', help='the bot project name')
-    parser.add_argument('directory', help='the directory to place it in (default: .)', nargs='?', default=Path.cwd())
-    parser.add_argument('--prefix', help='the bot prefix (default: $)', default='$', metavar='<prefix>')
-    parser.add_argument('--sharded', help='whether to use AutoShardedBot', action='store_true')
-    parser.add_argument('--no-git', help='do not create a .gitignore file', action='store_true', dest='no_git')
+    parser.add_argument("name", help="the bot project name")
```

Comment:
This also seems worse reading wise

Reply:
Overall, these are 2 tiny things, but highlight a bigger issue, they make the code worse to read, and thus need to be handled with ignore/skip statements, so we can either add them, which will take a while, or let them be worse reading wise